plugin/cache: add a new keepttl option (#5879)
miekg/coredns: plugin/cache/README.md

  greater than 5 minutes.
* `disable`  disable the success or denial cache for the listed **ZONES**.  If no **ZONES** are given, the specified
  cache will be disabled for all zones.

> * `keepttl` enforce the cache plugin to not reduce the TTL he got from its backend. This can be useful

Reviewer: ```suggestion
* `keepttl`  do not age the TTL when serving responses from cache. The entry will still be removed from cache when the TTL expires as normal, but until it expires responses will include the original TTL instead of the remaining TTL. This can be useful 
```

  if CoreDNS is used as an authoritative server and you want to serve a consistent TTL to the clients.

>   This is **NOT** recommended when CoreDNS is not an authoritative server as using this option could result

Reviewer: ```suggestion
  This is **NOT** recommended when CoreDNS is caching records it is not authoritative for because it could result
```

Reviewer: Suggesting this because CoreDNS can be hybrid authoritative for some zones, and non-authoritative others.

  greater than 5 minutes.
* `disable`  disable the success or denial cache for the listed **ZONES**.  If no **ZONES** are given, the specified
  cache will be disabled for all zones.
* `keepttl` do not age TTL when serving responses from cache. The entry will still be removed from cache

>   when the TTL expires as normal, but until it expires responses will include the responses will include the

Reviewer: ```suggestion
  when the TTL expires as normal, but until it expires responses will include the
```

Author: Oups my bad indeed sorry for that
